source/_integrations/nuki.markdown

```diff
@@ -71,7 +71,15 @@ Events generated by Nuki are sent as events of type `nuki_event` with the follow
 | `type`               | The type of the event. Values: `ring`
 | `entity_id`          | The ID of the entity generating the event.
 
+## Homekit support
+
+The 2nd and 3rd Generation Smart Locks support HomeKit via Bluetooth and can directly integrate with Home Assistant.
+
 ## MQTT support
 
-The Nuki Smart Lock 3.0 Pro also [supports MQTT](https://support.nuki.io/hc/en-us/articles/12947926779409-MQTT-support) and can directly integrate with Home Assistant through [MQTT discovery](/integrations/mqtt/#mqtt-discovery).
+The Nuki Smart Lock 3.0 Pro and 4th generation devices also [supports MQTT]([https://support.nuki.io/hc/en-us/articles/12947926779409-MQTT-support]) and can directly integrate with Home Assistant through [MQTT discovery](/integrations/mqtt/#mqtt-discovery).
 Specific information can be found [here](https://developer.nuki.io/t/mqtt-api-specification-v1-3/17626).
+
+## Matter Support 
+
+Smart Lock Ultra, Smart Lock Pro, Smart Lock Go, Smart Locks of 4th Generation support Matter over Thread and can directly integrate with Home Assistant.
```

Comment:
Instead of sprinkling this document with all kind of references to different ways of integrating; I'd suggest to make a single paragraph that explains it all for all of them.

Reply:
I'll update it. 
Let me try to figure out how to do this in GitHub. 
Apologies if I hit the wrong button somewhere.

Reply:
I made some updates. I hope it shows up correctly...